docs/testplan/smart-switch/high-availability/keysight_ha_testplan.md

```diff
@@ -0,0 +1,259 @@
+#### Test Objective 2
+
+![HA Perfect Sync Between HA Sets](images/ha_test_perfect_sync.svg)
+#### Steps for Test Case 2
+* Configure 2 DPUs networked across two SmartSwitches forming an HA set.
+* The SmartSwitch configuration will consist of 2 DPUs sharing the same network configurations
+such as: IPs, MACs, VLan/VxLan, ENIs.
+* The UHD will be configured to split traffic into two separate entities.
+* There will be a physical link between front panel ports of both DPUs.
+* Verify links are up and start all protocols and verify traffic is established.
+* Enable csv logging and check the state of the DPUs through the API.
+* Apply and start traffic stateful and stateless.
+* Verify traffic is flowing without any loss.
+* Through mgmt switch traffic from the Active to Standby node.
+* Mark start time at beginning of test as DPU is removed from topology.
+* The Backup should become Active immediately.
+* Mark time when the Standby DPU becomes Active and fully running traffic.
+* Perform perfect sync between DPUs, flow tables shall sync, verify success by observing state
+sync state.
+* Using traffic generator tools to verify metrics associated with number of Concurrent
+Connections, Connection rate, and TCP/UDP failures collect data before, during and after
+switchover.
+
+### Test case3 Link Loss HA Set
+#### Test Objective 3
+In this scenario we will test the switchover between an Active DPU to it's paired Standby DPU.
+
+![HA LinkLoss](images/ha_linkloss_test.svg)
+#### Steps for Test Case 3
+* Configure 2 DPUs networked within separate SmartSwitches forming a HA set between DPU0 and DPU0.
+* The SmartSwitch configuration will consist of 2 DPUs sharing the same network configurations
+such as: IPs, MACs, VLan/VxLan, ENIs.
+* The UHD will be configured to split traffic into two separate entities.
+* There will be a physical link between front panel ports of both DPUs.
+* Verify links are up and start all protocols and verify traffic is established.
+* Enable csv logging and check the state of the DPUs through the API.
+* Apply and start traffic stateful and stateless.
+* Verify traffic is flowing without any loss.
+* Through mgmt port remove link connection between DPU0 of SmartSwitch0.
+* Mark start time at beginning of test as link connection is removed.
+* There should be 100% link failure with the Active DPU during switchover.
+* DPU1 shall become the new Active DPU in the HA set.
+* Mark time when the Standby DPU becomes Active and fully running traffic.
+* Measure convergence time from start of the link removal between SmartSwitch0 and SmartSwitch1
+switchover.
+* Using traffic generator tools to verify metrics associated with number of Concurrent
+Connections, Connection rate, and TCP/UDP failures collect data before, during and after
+switchover.
+
+### Test case4 Link Loss Multiple HA Sets
+#### Test Objective 4
+In this scenario we will test the switchover between an Active DPU to it's paired DPU from a
```

Comment:
The objective doesn't contain the goal we are trying to achieve with multiple sets. It will be better to have it clarified.